sierra2mlir/src/statements/inline_jumps.rs

```diff
@@ -427,8 +427,20 @@ impl<'ctx> Compiler<'ctx> {
             // its safe if new_length is 0
             let new_length_zext_op = self.op_zext(&block_pop_idx, new_length, self.u64_type());
             let new_length_zext = new_length_zext_op.result(0)?.into();
+
+            let element_size_bytes = element_type.get_width() / 8;
```

Comment:
I think this might fail if we ever decide to change felts to `i252`.
```suggestion
            let element_size_bytes = (element_type.get_width() + 7) / 8;
```